docs/cedar.md

```diff
@@ -45,12 +45,20 @@ The calls that are made to the CEDAR APIs are defined by code generated by `go-s
 The Go code is generated by a tool called `go-swagger`. This is a different tool from the Go generator in swagger codegen, and is a standalone tool.
 [You can view the installation documentation here](https://goswagger.io/install.html).
 
-This tool is not executed as part of any build process, and should be run manually when new swagger files are added.
+For CEDAR Core, this tool is not executed as part of any build process, and should be run manually when new swagger files are added. From the `pkg/cedar/core` directory:
 
 ```terminal
-swagger generate client -f cedar_intake.yml -c ./gen/client -m ./gen/models
+swagger generate client -f cedar_core.yml -c ./gen/client -m ./gen/models
 ```
 
+CEDAR LDAP is similar, just in the `pkg/cedar/cedarldap` directory:
+
+```terminal
+swagger generate client -f cedar_core.yml -c ./gen/client -m ./gen/models
```

Comment:
Pretty minor, but guessing ldap needs slightly different instructions? (This is minor - we may even replace CEDAR LDAP soon enough 😅 )

Reply:
Good catch, fixed.